docs/zkapps/o1js/custom-tokens.mdx

```diff
@@ -12,200 +12,106 @@ keywords:
   - token id
   - token accounts
   - token owner
+  - tokens
   - zkApps updates
+  - tokencontract
 ---
 
-:::info
+# Custom Tokens
 
-zkApp programmability is not yet available on the Mina Mainnet, but zkApps can now be deployed on Berkeley Testnet.
+Blockchain applications have various use cases for custom tokens, including a real-world financial asset, stake in an on-chain protocol, or even skill points in a game.
 
-:::
+Most blockchains, like Ethereum, do not natively support custom tokens. You implement custom tokens as smart contracts on top of the execution layer of the underlying protocol.
+Token standards ensure the interoperability of applications on Etherum, these standardisations are agree upon in ERCs, Ethereum Request for customElements, such as the fungible token standard [ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/).
+The Ethereum community has created and agreed upon other reference implementations and standardisation that are audited and easy to configure, such as [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/) for NFTs. 
 
-# Custom Token API
+Mina supports custom token functionality at a low level in the tech stack. Mina treats custom tokens almost the same way as the native MINA token. This approach offers the following benefits:
 
-You can use o1js to perform common token operations, such as minting, burning, and sending tokens.
+- As a developer, you do not have to manage as many boilerplate contracts.
+- Developers don't need to keep track of accounts and balances themselves.
+- It is more secure because fewer vulnerabilities can result from incorrect configuration and deployment.
+- Interacting with custom tokens is more efficient: End users experience lower transaction fees because of less computation at the execution layer. 
```

Comment:
I rephrased this. Not sure if we want to keep it -- tokens dont directly impact this, but an argument can be made that it does because Ethereums token are fee hungry, and ours arent

Reply:
deleting, good call